src/features/hotspots/root/HotspotsList.tsx

```diff
@@ -1,21 +1,35 @@
 import React, { memo, useCallback, useMemo } from 'react'
-import { BottomSheetSectionList } from '@gorhom/bottom-sheet'
+import { SectionList } from 'react-native'
```

Comment:
Does this need to be imported from `react-native-gesture-handler`?

Reply:
Ah its not in a bottom sheet anymore right? So probably not then.

Reply:
Yeah, right, it's just a regular View now.